main.tf

```diff
@@ -543,16 +543,8 @@ data "aws_iam_policy_document" "eks_cw_iam_role_policy" {
 
   statement {
     actions = [
-      "firehose:CreateDeliveryStream",
-      "firehose:DeleteDeliveryStream",
-      "firehose:DescribeDeliveryStream",
       "firehose:PutRecord",
-      "firehose:PutRecordBatch",
-      "firehose:StartDeliveryStreamEncryption",
-      "firehose:StopDeliveryStreamEncryption",
-      "firehose:TagDeliveryStream",
-      "firehose:UntagDeliveryStream",
-      "firehose:ListTagsForDeliveryStream"
+      "firehose:PutRecordBatch"
     ]
     effect    = "Allow"
     resources = ["arn:aws:firehose:*:${data.aws_caller_identity.current.account_id}:deliverystream/${local.firehose_delivery_stream_name}"]
```

Comment:
This will apply these privileges for the Firehose Stream that is created, when I look at the link ticket this is based off https://lacework.atlassian.net/browse/LINK-1468 then I see that the intention was to only apply this to this particular customers `FirehoseARN` that they supply - is this not the case anymore? This change will apply to all users, is that what the intention is?

Reply:
Yes, the LINK, although it's filed for 1 customer, requests us to do this for all customers